laravel-app/app/Services/CommentService.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace App\Services;
+
+use App\Http\Requests\CommentRequest;
+use App\Models\PostComment;
+use App\Models\UserPost;
+
+class CommentService
+{
+    /**
+     * Logic for storing comment
+     */
+    public function storeComment(CommentRequest $request, UserPost $post): void
```

Comment:
Why does this method return void? What if I want to get the created comment and display it to my controller?

Reply:
Yes sir, will resolve it 🙇